torch/testing/_internal/common_nn.py

```diff
@@ -364,7 +458,10 @@ def bce_with_logistic_no_reduce_test():
         fullname='BCEWithLogitsLoss_no_reduce',
         constructor=wrap_functional(
             lambda i: F.binary_cross_entropy_with_logits(i, t.type_as(i), reduction='none')),
+        cpp_function_call='''F::binary_cross_entropy_with_logits(
+            i, t.to(i.options()), F::BinaryCrossEntropyWithLogitsFuncOptions().reduction(torch::kNone))''',
         input_fn=lambda: torch.rand(15, 10).clamp_(2.8e-2, 1 - 2.8e-2),
+        cpp_var_map={'i': 'input', 't': t},
```

Comment:
It has bugged me a little while reading this that we are using the string "input" as a special value that means "ask the module test for inputs, and pass those in". On the other hand, I don't know what else would be better

Reply:
I think an alternative is to require people to create the `input` tensor outside of the test dict, and do `input=the_input_tensor` in the test dict. But I am not sure if that would impose too much restriction on how people write the tests

Reply:
I think I would feel better about the magic here if we changed the magic string to make it more obvious that it's a magic string. Something like 'get_input()'

Reply:
Or if we passed in a global from somewhere, cpp_var_map={'i': GET_INPUT, 't': t},

Reply:
Thanks for the suggestions! I changed it to `"_get_input()"`, to look exactly like the method we are calling on the test instance